_build_cfg.yml

```diff
@@ -218,6 +218,15 @@ Topics:
         File: dynamically_provisioning_pvs
       - Name: Volume Security
         File: pod_security_context
+      - Name: Persistent Storage Troubleshooting
+        File: storage_troubleshooting
+  - Name: Persistent Storage Examples
+    Dir: storage_examples
+    Topics:
+      - Name: Overview
+        File: index
+      - Name: Sharing an NFS mount across two persistent volume claims
+        File: shared_nfs_e2e
```

Comment:
I believe the Travis build failed because this was not updated yet to the new `shared_storage` file name.